client/src/components/ProTx/components/charts/MainChart.js

```diff
@@ -2,24 +2,13 @@ import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { LoadingSpinner } from '_common';
-import {
-  getMaltreatmentTypeNames,
-  getMaltreatmentSelectedValues,
-  getMaltreatmentAggregatedValue,
-  getMaltreatmentTypesDataObject,
-  getPredictiveFeaturesDataObject
-} from '../shared/dataUtils';
-import {
-  plotConfig,
-  getPlotLayout,
-  getPlotDataBars
-} from '../shared/plotUtils';
 import ChartInstructions from './ChartInstructions';
 import DemographicsDetails from './DemographicsDetails';
 import MaltreatmentDetails from './MaltreatmentDetails';
 import PredictiveFeaturesTable from './PredictiveFeaturesTable';
 import AnalyticsDetails from './AnalyticsDetails';
```

Comment:
looks like there is a direct [commit](0567c6be1d31b2cf523af95b6b479cb42253c1c9) to `main` might have brought some of this PR already into main.